pkgs/applications/audio/mpc/default.nix

```diff
@@ -20,6 +21,14 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-2FjYBfak0IjibuU+CNQ0y9Ei8hTZhynS/BK2DNerhVw=";
   };
 
+  patches = [
+    # fix the build with meson 0.60 (https://github.com/MusicPlayerDaemon/mpc/pull/76)
+    (fetchpatch {
+      url = "https://github.com/MusicPlayerDaemon/mpc/commit/b656ca4b6c2a0d5b6cebd7f7daa679352f664e0e.patch";
+      sha256 = "1wi6lb5j06413l90w744gv3k1kv2w74ywswzfj1d5m1m268iqamy";
```

Comment:
Wrong hash.  Please, you can use plain `nix-prefetch-url` to get hashes for `fetchpatch`.  The point of the specialized function is that dynamically generated patches change slightly over time, so there's some post-processing that changes the hash and makes it stable.

Reply:
(I'm sorry, for the first time I wrote my sentence with reversed meaning. It's fixed now.)

Reply:
Sorry, how do I get the hash then?

Reply:
Figured it out, `fakeSha256`

Reply:
> Sorry, how do I get the hash then?

plug in lib.fakeSha256, try to build the package and get the right hash from the error message

Reply:
All done

Reply:
For reference, `nix-prefetch-url -A some.attribute.path` does work for this, avoiding e.g. the redownload, but unfortunately I'm not able to use it with lists... so one would have e.g. temporarily write it like
```
  foo =
    (fetchpatch {
      url = "https://github.com/MusicPlayerDaemon/mpc/commit/b656ca4b6c2a0d5b6cebd7f7daa679352f664e0e.patch";
    })
    ;
```
and `nix-prefetch-url -A somePackage.foo`